server/reload_test.go

```diff
@@ -33,6 +35,7 @@ func TestConfigReloadNoConfigFile(t *testing.T) {
 func TestConfigReloadUnsupported(t *testing.T) {
 	server, opts, config := newServerWithSymlinkConfig(t, "tmp.conf", "./configs/reload/test.conf")
 	defer os.Remove(config)
+	defer server.Shutdown()
```

Comment:
Is this needed since the server isn't being started? (note we're calling `newServerWithSymlinkConfig` and not `runServerWithSymlinkConfig`).